test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1006
+-- Description: PoliciesManager must allow all requested params in case "parameters" field is omitted
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- In case:
+-- 1) Send any single UI-related RPC , UI interface is not supported by the system
+-- 2) SDL receives UI.IsReady (available=false) from HMI
+-- Expected result:
+-- 1) SDL must respond "UNSUPPORTED_RESOURCE, success=false, info: UI is not supported by system" to mobile app
+-- Actual result:
+-- SDL responds with GENERIC_ERROR, success=false (logs are in attachment)
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local hmi_values = require("user_modules/hmi_values")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local function getHMIValues()
+  local params = hmi_values.getDefaultHMITable()
+  params.UI.IsReady.params.available = false
+  params.UI.GetCapabilities = nil
+  params.UI.GetLanguage = nil
+  params.UI.GetSupportedLanguages = nil
+  return params
+end
+
+local requestParams = {
+  cmdID = 11,
+  menuParams = {
+    position = 0,
+    menuName ="Commandpositive"
+  },
+  vrCommands = {
+    "VRCommandonepositive",
+    "VRCommandonepositivedouble"
+  },
+  grammarID = 1
+}
+
+local responseVrParams = {
+  cmdID = requestParams.cmdID,
+  type = "Command",
+  vrCommands = requestParams.vrCommands
+}
+
+local allParams = {
+  requestParams = requestParams,
+  responseVrParams = responseVrParams
+}
+--[[ Local Functions ]]
+local function sendAddCommand(params)
+  local cid = common.getMobileSession():SendRPC("AddCommand", params.requestParams)
+  common.getHMIConnection():ExpectRequest("VR.AddCommand", params.responseVrParams)
```

Comment:
Please add the expectation of the UI.AddCommand with Times(0)

Reply:
@GetmanetsIrina You can find the updates in the commit ac51975